### PR TITLE
[website] Add books to Resources page

### PR DESCRIPTION
**Pull request by tuhaihe**
### Motivation

Many books on Apache Pulsar have been published, so I would love to add the book resource to the website to help community members to learn Apache Pulsar.

### Modifications

Add books resources to the webpage.

### Documentation

- [x] no-need-doc

### CODE REVIEW

File: site2/website/data/resources.js [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+      author: "Jowanza Joseph",
+      link: 'https://www.oreilly.com/library/view/mastering-apache-pulsar/9781492084891/'      
+    },
+    {
+      book: 'Cloud-Native Microservices with Apache Pulsar',
+      author: "Rahul Sharma, Mohammad Atyab",
+      link: 'https://link.springer.com/book/10.1007/978-1-4842-7839-0'      
+    },
+    {
+      book: '深入解析 Apache Pulsar（Chinese Book)',
```
Comment (Anonymitaet) [inline site2/website/data/resources.js:19]:
```suggestion
      book: '深入解析 Apache Pulsar (Chinese Book, which means "Deep Dive into Apache Pulsar")',
```

**Comment by Anonymitaet** [comment]
@tuhaihe thanks for your contribution! 

1. Have you run a local preview before submitting changes? If so, can you please provide screenshots of your changes in the PR description? So that others know your changes clearly.
Steps of running a local preview: https://github.com/apache/pulsar/tree/master/site2#website 



3. You delete the backticks (``) so that Bot can not recognize the info and then labels the PR with `doc-info-missing`.
[x] `doc-required`  ✅
[x] doc-required ❌
Instrctions of doc labels: https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0/edit

File: site2/website/data/resources.js [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+      book: 'Cloud-Native Microservices with Apache Pulsar',
+      author: "Rahul Sharma, Mohammad Atyab",
+      link: 'https://link.springer.com/book/10.1007/978-1-4842-7839-0'      
+    },
+    {
+      book: '深入解析 Apache Pulsar（Chinese Book, which means "Deep Dive into Apache Pulsar")',
+      author: "Lin Lin",
+      link: 'https://www.phei.com.cn/module/goods/wssd_content.jsp?bookid=59310'      
+    }
+  ]
```
Comment (urfreespace) [inline site2/website/data/resources.js:23]: @tuhaihe Missing a comma here

**Comment by tuhaihe** [comment]
Hi @Anonymitaet @urfreespace Thanks for your guys's help. I update this PR and run it successfully in local. This is the screenshot. PTAL.

image

**Review by Anonymitaet — APPROVED** [review]

**Review by Shoothzj — APPROVED** [review]

**Review by Shoothzj — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**